kolibri/core/templates/kolibri/base.html

```diff
@@ -45,6 +46,10 @@
   </script>
 </rootvue>
 {% block frontend_assets %}
+<script>
+  // Set global language variable.
+  var languageCode = "{{ LANGUAGE_CODE }}";
```

Comment:
Would this potentially make testing harder, because the JS files now depend on a global variable set in HTML?